includes/model/class-comment.php

```diff
@@ -0,0 +1,291 @@
+<?php
+namespace Activitypub\Model;
+
+/**
+ * ActivityPub Comment Class
+ *
+ * @author Django Doucet
+ */
+class Comment {
+	private $comment;
+	private $id;
+	private $comment_author_url;
+	private $post_author;
+	private $in_reply_to;
+	private $content_warning;
+	private $permalink;
+	private $context;
+	private $to_recipients;
+	private $tags;
+	private $update;
+	private $deleted;
+	private $replies;
+
+	/**
+	 * Initialize the class
+	 */
+	public function __construct( $comment = null ) {
+		$this->comment 			  = $comment;
+		$this->id 				  = $this->generate_comment_id();
```

Comment:
Sorry for my very unuseful comment: but probably here it should be aligned with spaces instead of tabs

(I love indenting with tabs and aligning with spaces ihih)